ownCloud/Client/Actions/Actions+Extensions/DisplayExifMetadataAction.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  DisplayExifMetadataAction.swift
+//  ownCloud
+//
+//  Created by Michael Neuwert on 29.06.20.
+//  Copyright © 2020 ownCloud GmbH. All rights reserved.
+//
+
+/*
+* Copyright (C) 2020, ownCloud GmbH.
+*
+* This code is covered by the GNU Public License Version 3.
+*
+* For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+* You should have received a copy of this license along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+*
+*/
+
+import ownCloudSDK
+
+class DisplayExifMetadataAction : Action {
+	override class var identifier : OCExtensionIdentifier? { return OCExtensionIdentifier("com.owncloud.action.show-exif") }
+	override class var category : ActionCategory? { return .normal }
+	override class var name : String? { return "Image Metadata".localized }
+	override class var locations : [OCExtensionLocationIdentifier]? { return [.moreItem, .moreFolder, .contextMenuItem] }
+	class var supportedMimeTypes : [String] { return ["image"] }
+	override class var licenseRequirements: LicenseRequirements? { return LicenseRequirements(feature: .photoProFeatures) }
+
+	// MARK: - Extension matching
+	override class func applicablePosition(forContext: ActionContext) -> ActionPosition {
+		if forContext.items.count == 1, let item = forContext.items.first, item.type == .file, let mimeType = item.mimeType {
+			if supportedMimeTypes.filter({ mimeType.contains($0) }).count > 0 {
```

Comment:
Is this action available for all image types (gif, png, svg) or only for (jpeg, heic, raw) types?

Reply:
@hosy Excluded SVG and GIF which indeed don't make sense here